shanti/exercise.16.js

```diff
@@ -0,0 +1,33 @@
+// 16-PB-datastructure-array-III
+let myArr = [
+    'item1',
+    'item2',
+    [
+    2,
+    3,
+    4,
+        [
+            'subItem1','subItem2',
+            [
+                'subSubItem1', 'subSubItem2'
+            ]
+        ]
+   ]
+];
+
+/* 1. clone the array. */
+let arr2 = JSON.stringify(myArr);
```

Comment:
to clone the array we should do JSON.stringify(JSON.parse(myArr))

but we use this way with very advance level